modules/restore-replace-crashlooping-etcd-member.adoc

```diff
@@ -0,0 +1,125 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/replacing-unhealthy-etcd-member.adoc
+
+[id="restore-replace-crashlooping-etcd-member_{context}"]
+= Replacing a crashlooping etcd member
+
+You can replace an etcd member that is running, but is crashlooping.
+
+.Prerequisites
+
+* You have identified the unhealthy etcd member.
+* You have verified that the node is running, but etcd is crashlooping.
+* You have access to the cluster as a user with the cluster-admin role.
+* You have taken an etcd backup.
++
+[IMPORTANT]
+====
+It is important to take an etcd backup before performing this procedure, so that your cluster can be restored if you encounter any issues.
+====
+
+.Procedure
+
+. Stop the crashlooping etcd Pod.
+
+.. Debug the node that is crashlooping.
++
+In a terminal that has access to the cluster as a `cluster-admin` user, run the following command:
++
+----
+$ oc debug node/ip-10-0-131-183.ec2.internal <1>
+----
+<1> Replace this with the name of the unhealthy node.
+
+.. Change your root directory to the host:
++
+----
+sh-4.2# chroot /host
+----
+
+.. Move the existing etcd Pod file out of the Kubelet manifest directory:
++
+----
+sh-4.4# mv /etc/kubernetes/manifests/etcd-pod.yaml /var/lib/etcd-backup/
+----
++
+// TODO: Confirm sh-4.4# prompt is okay.
```

Comment:
@alaypatel07 When I tested, the prompt was 4.2 after you oc debug, then changed to 4.4 after you run `chroot /host`. Just wanted to verify with you that that's correct as shown here.